### PR TITLE
IDM docs

### PR DESCRIPTION
**Pull request by dsajdak**
documentation for IDM portal, creating accounts, 2FA management, getting started page, new index page

### CODE REVIEW

File: pages/2fa.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
-    write me
+## Enabling 2FA on your CCR account  
+[Follow these directions to properly enable 2FA on your CCR account](portals/idm/#enabling-two-factor-authentication)
+
+!!! Warning  
+    Your UBIT account in Duo is NOT tied to your CCR account.  You will NOT receive push notifications from your CCR account  
+
+## Logging into your CCR account with 2FA  
+
+[See these instructions for logging into CCR's portals with 2FA enabled](portals/idm/#logging-in-with-two-factor-authentication)
```
Comment (dsajdak) [inline pages/2fa.md:13]: this link doesn't work.  I'm fixing it in another PR

File: pages/2fa.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+
+Watch this super short video that demonstrates the login process:  
+![type:video](https://youtube.com/embed/g6hWYooFKWE)  
+
+## My 2FA Doesn't Work!  
+
+If you enabled 2FA prior to January 2023, it is possible your CCR account is not properly linked to whichever authentication app you're using.  Sometimes people forget to scan the QR code to link their account or some misunderstood and thought their CCR account was linked to their UBIT account.  If this is the case for you, you are locked out of your account.  Please contact CCR help to receive instructions on how to proceed.  For those enabling 2FA after the IDM portal update in January 2023, it will be more difficult to get in this locked account situation but if your OTP token isn't working or you lost access to the device that generates your OTP tokens, contact us!
+
+!!! Tip  
+    Changing phones? If you won't have access to both devices, disable 2FA on your CCR account prior to swapping phones. When you get your new phone [follow these instructions](portals/idm/#managing-tokens-for-devices) to link a new device and then re-enable 2FA
```
Comment (aebruno) [inline pages/2fa.md:23]: Fix page link, should be: `[See these instructions for logging into CCR's portals with 2FA enabled](portals/idm.md#logging-in-with-two-factor-authentication)`

File: pages/getting-access.md [hunk 3]
```diff
@@ -1,15 +1,23 @@
...
+- [Creating a CCR user account](#creating-a-ccr-user-account)
+- [Get added to an active allocation for the resource(s) you want to use](#requesting-an-allocation-in-coldfront)
+
+!!! Warning  
+    Access to most CCR resources is restricted to UB and Roswell Park networks (either on campus or connected to their VPN services).  Users without access to UB or RPCI accounts should contact CCR Help for VPN access.
 
 ## Creating a CCR user account
 
-TODO
+Users with UB or Roswell Park email addresses can create themselves a CCR user account following [these directions](../portals/idm/#create-a-ccr-system-account).  Once your account is created, make sure to [follow the instructions for enabling two factor authentication](../portals/idm/#managing-two-factor-authentication).  Now it's time to request access to the resources or services you want to use.
```
Comment (aebruno) [inline pages/getting-access.md:14]: Fix page link, should be: `[these directions](portals/idm.md#create-a-ccr-system-account)` and `[follow the instructions for enabling two factor authentication](portals/idm.md#managing-two-factor-authentication)`

File: pages/getting-access.md [hunk 4]
```diff
@@ -1,15 +1,23 @@
...
 
 ## Requesting an Allocation in ColdFront
 
-TODO
+ColdFront is CCR's resource and allocation management system.  ColdFront manages all of CCR's resources and the access all users have to those resources.  Faculty members (Principal investigators) must create themselves a project in ColdFront, request allocations for the resources their group wants to use, and then add their group members to their project.  Once an allocation is activated, users on that allocation will be able to access the resource.  [More details about ColdFront can be found here](../portals/coldfront).
```
Comment (aebruno) [inline pages/getting-access.md:18]: Fix page link, should be: `[More details about ColdFront can be found here](portals/coldfront.md)`

File: pages/getting-started.md [hunk 5]
```diff
@@ -15,18 +15,20 @@ This 20 minute presentation given by the CCR Director in 2020 provides a succinc
...
      - [Getting Access](getting-access.md)  
      - [Frequently Asked Questions about CCR](faq.md)  
-- If you're an experienced HPC user and ready to log onto a cluster, you probably want to know:
+- If you're an **experienced HPC user** and ready to log onto a cluster, you probably want to know:
      - What [clusters](hpc/node-types.md) are available  
-     - What [software is available](software/about.md) and how [environment modules](software/modules.md) work  
-     - How to [submit jobs](hpc/jobs.md)  
-     - How the [file system](hpc/storage.md) is organized
-- If you're new to HPC or would like additional technical background information, you can  
+     - Get instructions for how to [connect to our HPC systems with SSH](../hpc/login)
```
Comment (aebruno) [inline pages/getting-started.md:20]: Fix page links.. need to link to md file, for example: `hpc/storage.md`

File: pages/getting-started.md [hunk 6]
```diff
@@ -37,25 +39,25 @@ This 20 minute presentation given by the CCR Director in 2020 provides a succinc
...
-- Examples of cloud uses: database warehouses, websites, container orchestration, software development projects, and other non-HPC scientific applications  
+- Researchers utilize the on-premise cloud for projects not suited for the HPC environment  
+- Examples of cloud uses: database warehouses, websites, container orchestration, software development projects, proof-of-concept testing, and other non-HPC scientific applications  
 - Cloud compute time is billed by CPU hour & storage is sold in increments of 1TB  
-- Billing info can be found [here](lake-effect.md)  
+- Billing info can be found [here](../cloud/lake-effect)  
 
 !!! Note  
-    You must be on an active allocation for the resource(s) you want to use.  [More info here](getting_access.md)  
+    You must be on an active allocation for the resource(s) you want to use.  [More info here](../getting_access)
```
Comment (aebruno) [inline pages/getting-started.md:57]: This link is not correct. Should link to md file

File: pages/hpc/login.md [hunk 7]
```diff
@@ -1,103 +1,144 @@
...
-- An active allocation in ColdFront
-- Two-factor authentication
-- An ssh client
+To get started with using CCR's clusters and shared storage you need the following:
 
-Users accessing CCR's resources will be connected to a login node. A login node
-is a outward facing node within CCR that users can connect to from their local
-machines. Once on a login node, users can perform a limited number of tasks:
+- A CCR user account - Don't have one?  [See here](../getting-access)  
+- An active allocation for the resource in ColdFront - No allocation?  [See here](getting-access/#requesting-an-allocation-in-coldfront)
```
Comment (aebruno) [inline pages/hpc/login.md:8]: Fix link

File: pages/hpc/login.md [hunk 8]
```diff
@@ -1,103 +1,144 @@
...
 
-!!! note
-    The login node policy states that login nodes should not be used for
-    resource-intensive tasks such as running code. For all other tasks, users
-    should run batch jobs, interactive jobs, or use the compile nodes.
+## Getting an account & access  
 
-![type:video](https://www.youtube.com/embed/g6hWYooFKWE)
+UB faculty, staff, students, those with a UB volunteer appointment, and those with Roswell Park email addresses, can create their own CCR account using the CCR identity
+management portal, [following these instructions](portals/idm/#create-a-ccr-system-account).  Once an account is created, users must be added to an active allocation for one of the CCR clusters in order to access a login node.  Access is managed in CCR's resource and allocation management system, [ColdFront](https://coldfront.ccr.buffalo.edu).  Please refer to [these instructions for more information](getting-access/#requesting-an-allocation-in-coldfront).
```
Comment (aebruno) [inline pages/hpc/login.md:18]: Fix link

**Review by aebruno — CHANGES_REQUESTED** [review]
Content looks great! Found a bunch of invalid links however. When linking to a page you should link to the markdown file. [See here](https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages) for more info on page linking in mkdocs. 

Looks like anywhere you link to a page you need to fix. Added a few comments on how to fix but will need to update all page links. For example:

This:
 ` [See here](getting-access/#requesting-an-allocation-in-coldfront)`

Should be:

`[See here](getting-access.md#requesting-an-allocation-in-coldfront)`

File: pages/index.md [hunk 9]
```diff
@@ -1,22 +1,37 @@
...
+    - Other portals & services: see portal menu on the left  
+- **Research Computing (HPC) Specific Topics:**  
+    - [Batch Job System](hpc/jobs.md)  
+    - [Data Transfer Options](hpc/data-transfer.md)  
+    - [Linux and Slurm Cheat Sheet](https://buffalo.box.com/s/nqj3neyt2w1dtb3gix6zxqx5gcc9x30n)  
+    - [Software Module System](software/modules.md)  
+    - [Compiling Software](software/building.md)  
+    - [Visualization Servers](hpc/viz.md)  
+- **Research Cloud Specific Topics:**  
+    - [LakeEffect Cloud Quick Start](cloud/lake-effect)
```
Comment (aebruno) [inline pages/index.md:29]: These links need fixing.. missing *.md  should be `cloud/lake-effect.md` .. etc.

File: mkdocs.yml [hunk 10]
```diff
@@ -39,11 +39,11 @@ markdown_extensions:
...
 nav:
   - Home: index.md
+  - Frequently Asked Questions: faq.md
   - Accessing CCR Resources:
     - Getting Access: getting-access.md
     - Getting Started: getting-started.md
     - Two-Factor Authentication: 2fa.md
-    - Frequently Asked Questions: faq.md
   - High Performance Computing:
     - Logging In: hpc/login.md
```
Comment (aebruno) [inline mkdocs.yml:48]: Let's add `hpc/viz.md` to the nav

**Review by aebruno — CHANGES_REQUESTED** [review]
Find a few minor tweaks

**Review by aebruno — APPROVED** [review]

**Pull request merged**